instrumentation/github.com/gin-gonic/gin/otelgin/test/gintrace_test.go

```diff
@@ -120,7 +120,6 @@ func TestError(t *testing.T) {
 	attr := span.Attributes()
 	assert.Contains(t, attr, attribute.String("net.host.name", "foobar"))
 	assert.Contains(t, attr, attribute.Int("http.status_code", http.StatusInternalServerError))
-	assert.Contains(t, attr, attribute.String("gin.errors", "Error #01: oh no\n"))
```

Comment:
This test already checks for errors. We can check it being recorded here, no need for a new test.
We can also add a check for empty `span.Events` in `TestTrace200`.